app/assets/stylesheets/responsive/_request_style.scss

```diff
@@ -117,14 +121,88 @@ a img.attachment_image {
   }
 }
 
+.correspondence_delivery--sent {
+  background-color: mix(#fff, $color-delivery-sent, 85%);
+}
+
+.correspondence_delivery--delivered {
+  background-color: mix(#fff, $color-delivery-delivered, 85%);
+}
+
+.correspondence_delivery--failed {
+  background-color: mix(#fff, $color-delivery-failed, 85%);
+}
+
 .toggle-delivery-log {
-  // Pre-load the spinner image by hiding it outside the element bounds
-  background: transparent image-url('loader-000-on-fff-16px.gif') -100px -100px no-repeat;
+  padding-right: 8px + 16px;
+  margin-right: 4px; // remember also ~4px from whitespace character after inline element
+  background: transparent none 100% 50% no-repeat;
+
+  // Preload the spinner image, because the ajax request
+  // will likely block it from loading when we actually need it.
+  &:after {
+    content: "";
+    display: block;
+    width: 0;
+    height: 0;
+    position: absolute;
+    top: 0;
+    left: 0;
+    background: transparent image-url('loader-000-on-fff-16px.gif') 0 0 no-repeat;
+    background-size: 16px 16px;
+  }
+}
+
+.toggle-delivery-log--sent {
+  background-image: image-url('delivery-status/sent.png');
```

Comment:
The question mark makes it look like we don't know what the app is doing. How about making this a blue tick?